tensorflow/java/src/main/java/org/tensorflow/op/core/Zeros.java

```diff
@@ -0,0 +1,54 @@
+package org.tensorflow.op.core;
```

Comment:
@karllessard : Could you add the Copyright notice header to this file (and other new files)?